src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -46,9 +46,9 @@ private static ArgumentParser getArgumentParser() {
                 .action(new HelpArgumentAction());
 
         mutexParser.addArgument("-config")
-                .type(Arguments.fileType().verifyExists().verifyIsFile().verifyCanRead())
+                .type(new ConfigFolderArgumentType())
                 .metavar("PATH")
-                .help("The path to the CSV config file to read.");
+                .help("The path to the config folder containing repo-config.csv.");
```

Comment:
`containing the config files.` in general, instead of specifically `repo-config.csv`, since we are adding more in the future.

Reply:
I intend to print the filename explicitly as RepoSense will fast-fail if otherwise.

Also, the upcoming specs will only add support for `repo-config.csv` and `author-config.csv`.

Reply:
>I intend to print the filename explicitly as RepoSense will fast-fail if otherwise.

fast-fail? We are just updating the help message here right?

Reply:
**Correction for clarity**: I intend to print the filename explicitly as RepoSense will fast-fail ~if otherwise~ as now the `-config` expects a folder that contains a `repo-config.csv`.